### PR TITLE
Small refactor

### PR DESCRIPTION
**Pull request by pbkompasz**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [X] Refactoring

## Context

- [X] Part of #6782
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [X] My contribution builds **clean without any errors or warnings**
- [X] I've merged recent default branch -- **main** and I've no conflicts
- [X] I've tried to respect high code quality standards
- [X] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [X] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=16hqk8Tjugo9AwSzKkxXc1dsRK7iJetpx1YBaoXYwdHzuNAt&usdamount=50&donation=true)

#### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### <samp>🤖 Generated by Copilot at 8cab8c8</samp>

This pull request refactors and simplifies some functions in the `composables/massmint/parsers`, `utils`, and `stores` modules. The main goal is to improve code quality, modularity, performance, and clarity.

### <samp>🤖 Generated by Copilot at 8cab8c8</samp>

> _We slice the history with our splicing blade_
> _We parse the text and process every block_
> _We execute the transactions with our hash of rage_
> _We refactor the code and unleash the metal shock_

### CODE REVIEW

File: composables/massmint/parsers/parseTxt.ts [hunk 1]
```diff
@@ -56,26 +56,46 @@ function parsePriceAndCurrency(fieldValue: string): {
...
+  const entry = processBlock(block)
+  const { $consola } = useNuxtApp()
+
+  if (entry.file) {
+    entries[entry.file] = {
+      file: entry.file,
+      name: entry.name || undefined,
+      description: entry.description || undefined,
+      price: entry.price || undefined,
+      currency: entry.currency || undefined,
```
Comment (floyd-li) [inline composables/massmint/parsers/parseTxt.ts:93]: seems `|| undefined` is not useful here,  can we remove it?

**Review by floyd-li — APPROVED** [review]
small stuff otherwise lgtm

**Comment by prury** [comment]
> @prury can have check on mass mint if I see right code part 👀

Massminted some stuff on RMRK, went ok ✔️ 
https://deploy-preview-6983--koda-canary.netlify.app/ksm/collection/7EA1DCF47E98A25067-MUSH
https://kusama.subscan.io/tx/0xeda75ea58a07417e3d73f862046631e2194e5f12536b21bf639b99a8b0473b77
https://kusama.subscan.io/tx/0xfc638009f11817ed214e9b057e8968601173d1a08b732e9c36a26351bac72610

**Comment by yangwao** [comment]
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 4.58 USD/DOT ~ 10.917 $DOT
🧗 [16hqk8Tjugo9AwSzKkxXc1dsRK7iJetpx1YBaoXYwdHzuNAt](https://polkadot.subscan.io/account/16hqk8Tjugo9AwSzKkxXc1dsRK7iJetpx1YBaoXYwdHzuNAt)
🔗 [0x2acb3ba7f681e6daa0a3c31d7f3206e323dfed3402ed616f98a247b8c937f141](https://polkadot.subscan.io/extrinsic/0x2acb3ba7f681e6daa0a3c31d7f3206e323dfed3402ed616f98a247b8c937f141)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**